### ISSUE TITLE
Gate::before ignored

### ISSUE THREAD

**Issue by hakimihamdan88** [spatie/laravel-permission#539]
im wondering why version 2.7.2 and above, will ignore 

``` 
Gate::before(function ($user, $ability) {

            if ($user->isAdmin()) {
                return true;
            }
        });

```

this code works fine on 2.7.0 and 2.7.1

**Comment by drbyte** [comment]
A pull request is in-progress to fix that: #538

**Comment by AlexVanderbist** [comment]
Fixed in v2.7.5!
